src/ui/src/core_components/other/CoreJsonViewer.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+	<div>
```

Comment:
As a convention we add a class to the root element i.e. `class="CoreJsonViewer"`